modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -12,6 +12,8 @@ import {getRandomInt} from '../../utils/getRandomInt';
 
 export const test = mergeTests(apiHelpersTest, loginTest(), objectPagesTest);
 
+const objectRelationshipRandomNumber = Math.floor(Math.random() * 99);
```

Comment:
Hi @rebsilva, Perhaps maintaining this same constant globally will influence someone in the future to use the same constant to create two relationships and face problems in the future, I would like to know what you think about this.

Reply:
i removed it and put `Math.floor(Math.random() * 99)` directly, i see no need to create a const or a function for this atm